pkg/sdk/go/certs.go

```diff
@@ -34,7 +35,7 @@ func (sdk mfSDK) IssueCert(thingID string, keyBits int, keyType, valid, token st
 	}
 
 	url := fmt.Sprintf("%s/%s", sdk.certsURL, certsEndpoint)
-	resp, err := request(http.MethodPost, token, url, d)
+	resp, err := request(http.MethodPost, apiutil.BearerPrefix+token, url, d)
```

Comment:
Why are you not using `sdk.processRequest` here?

Reply:
Updated to used `sdk.processRequest`